app/models/fire_cloud_client.rb

```diff
@@ -17,6 +17,8 @@ class FireCloudClient < Struct.new(:access_token, :api_root, :storage, :expires_
 	PORTAL_NAMESPACE = 'single-cell-portal'
 	# location of Google service account JSON (must be absolute path to file)
 	SERVICE_ACCOUNT_KEY = File.absolute_path(ENV['SERVICE_ACCOUNT_KEY'])
+	#location of Google service account JSON (must be absolute path to file)
```

Comment:
I think this may be a leftover from when firecloud crashed and I was trying to get things working. Removed